tidb-server/main.go

```diff
@@ -154,6 +156,13 @@ func main() {
 	os.Exit(0)
```

Comment:
`os.Exit(0)` will make no chance to execute `defer syncLog()` on my local test

Reply:
Done.